server/backend/testspg/router/test_router_application.py

```diff
@@ -244,4 +257,243 @@ def test_fam_application_endpoints_invlid_path_application_id_type(
         headers=jwt_utils.headers(token)
     )
     assert response.status_code == HTTPStatus.UNPROCESSABLE_ENTITY
-    assert invalid_path_router_msg in response.text
\ No newline at end of file
+    assert invalid_path_router_msg in response.text
+
+
+# ---------------- Test get application user role assignment filtering scenarios ------------ #
+
```

Comment:
Sorry @MCatherine1994 , there are lots of constants refactoring.
The real test is just below.
Thank you!!